demos/browser/package-lock.json

```diff
@@ -1,7204 +1,8 @@
 {
   "name": "amazon-chime-sdk-js-demo",
   "version": "2.0.0",
-  "lockfileVersion": 2,
```

Comment:
Please use the later versions of NPM. The`lockfileVersion` should remain the same.

Reply:
Sure, will do